app/controllers/qc_files_controller.rb

```diff
@@ -11,17 +11,20 @@ class QcFilesController < ApplicationController
   before_action :find_assets, only: %i[create index]
 
   def index
-    respond_to { |format| format.json { render json: { 'qc_files' => asset.qc_files } } }
+    respond_to { |format| format.json { render json: { 'qc_files' => asset.qc_files.map { |q| qc_file_to_json(q) } } } }
   end
 
   def show
-    response = api.qc_file.find(params[:id]).retrieve
-    filename = /filename="([^"]*)"/.match(response['Content-Disposition'])[1] || 'unnamed_file'
-    send_data(response.body, filename: filename, type: 'sequencescape/qc_file')
+    qc_file = Sequencescape::Api::V2::QcFile.find(uuid: params[:id]).first
```

Comment:
We could (in a separate piece of work) create an alias for the `Sequencescape::Api::V2` module.  Maybe something like `SSAPIv2` so that these would become `SSAPIv2::QcFile` etc.  I don't think we should name the real module differently, but an alias would be a reasonable approach.